### PR TITLE
.github/workflows: add labels sync

### PR DESCRIPTION
**Pull request by laurentsenta**
Contributes to #1511: clarify labels

### CODE REVIEW

**Review by galargh — APPROVED** [review]
Coooooool!

**Comment by laurentsenta** [comment]
The PR that removed it lack context but that was probably intended to simplify the workflow (since we used circleci at the time).
https://github.com/testground/testground/pull/1167

**Comment by brdji** [comment]
I'm out of the loop, what does this action do?
Sync Github labels with CircleCI?

**Comment by laurentsenta** [comment]
@brdji no worries, this PR syncs the `.github/label.yml` configuration with the repo (https://github.com/testground/testground/labels/) using a GitHub CI action.

**Review by brdji — APPROVED** [review]

**Comment by laurentsenta** [comment]
@galargh @brdji I'll merge this PR today, which will help with implementing the roadmap, etc.

This will remove the `Bloxico` label and `team/ipdx`, BUT I moved every ticket and tagged them in the project board, no information is lost:
https://github.com/orgs/testground/projects/2/views/1

File: .github/workflows/labels-sync.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      - ".github/workflows/labels-sync.yml"
+    branches:
+      - master
+jobs:
+  build:
+    name: Sync labels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: micnncim/action-label-syncer@v1.3.0
```
Comment (galargh) [inline .github/workflows/labels-sync.yml:15]: I'd pin this to a hash to make it more secure.

Comment (laurentsenta) [reply .github/workflows/labels-sync.yml]: Done!

File: .github/workflows/labels-sync.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      - ".github/workflows/labels-sync.yml"
+    branches:
+      - master
+jobs:
+  build:
+    name: Sync labels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: micnncim/action-label-syncer@3abd5ab72fda571e69fffd97bd4e0033dd5f495c
```
Comment (galargh) [inline .github/workflows/labels-sync.yml:15]:
```suggestion
      - uses: micnncim/action-label-syncer@3abd5ab72fda571e69fffd97bd4e0033dd5f495c # v1.3.0
```

**Pull request merged**